server/src/utils/fight/fightMethods.ts

```diff
@@ -981,15 +998,20 @@ const activateSuper = (
         // Change pet owner
         pet.master = fighter.id;
         pet.team = fighter.team;
+        // Lower pet initiative
+        pet.initiative -= 1;
       }
 
-      // Abort if no pet hypnotised
-      if (hypnotisedPets.length === 0) return false;
+      // Hypnotize opponent's brutes
+      const opponents = getOpponents({ fightData, fighter})
+        .filter((opponent) => opponent.type !== 'pet')
+      opponents.forEach((opponent) => opponent.hypnotized = true);
```

Comment:
forEach should never return a value
```suggestion
      opponents.forEach((opponent) => {
        opponent.hypnotized = true;
      });
```